app/templates/application.hbs

```diff
@@ -1,17 +1,17 @@
 <section class="application">
-  {{outlet}}
-
-  {{add-recipe}}
+  <h1>Super Sweet Recipe App</h1>
+    {{outlet}}
+  <!-- {{add-recipe}} -->
```

Comment:
Take out commented code. Also don't need this {{add-recipe}} in application, that's what {{outlet}} does for us.